tests/publiccloud/storage_perf.pm

```diff
@@ -120,6 +154,33 @@ sub run {
                 values => $values
             };
             $data = influxdb_push_data($url, 'publiccloud', $data);
+            my %influx_read_args = (
```

Comment:
I'm first of all sorry Anton that I'm late with such request so feel free to ignore it but didn't you think about creating separate test module for this analyzing code?

Reply:
no I can not say that you are too late , so from this perspective I don't see any issue in your request . And I think you at least partially right . I think it will be right idea to "pack"  this logic into dedicated function e.g. `analyse_previous_results` ( feel free to suggest name this one was generated within 5 seconds :) ) .
But I have concerns regarding separate module . I am using `$values` hash and I don't want to create copy of it in this separate module .
So do you think wrap this logic into a function will be good enough ? Or you want to continue conversation about separate module ? ( in such a case I am happy to hear ideas about `$values`  hash )

Reply:
Separate subroutine is OK and the name is also nice!

Reply:
yeah, not bad idea of separate method outside of `run` :) 
I also think separate module is not necessary.

Reply:
done